subprojects/core-api/src/main/java/org/gradle/api/artifacts/transform/TransformAction.java

```diff
@@ -25,16 +25,18 @@
  *
  * <p>Implementations can receive parameters by using annotated abstract getter methods.</p>
  *
- * <p>A property annotated with {@link TransformParameters} will receive the object provided by {@link ParameterizedTransformSpec#getParameters()}.
+ * <p>A property annotated with {@link InjectTransformParameters} will receive the object provided by {@link TransformSpec#getParameters()}.
```

Comment:
Aren't we adding this method to the interface? My understanding was that @adammurdoch's changes are already in to do this.

Reply:
I just merged Adam's changes, I'll add the method to the interface now.

Reply:
I added a getter to the interface.